libraries/fc/src/thread/context.hpp

```diff
@@ -14,6 +13,12 @@
 #define BOOST_COROUTINES_NO_DEPRECATION_WARNING // Boost 1.61
 #define BOOST_COROUTINE_NO_DEPRECATION_WARNING // Boost 1.62
 
+#if BOOST_VERSION >= 106800
+#include <boost/context/continuation_fcontext.hpp>
+#else
+#include <boost/context/all.hpp>
```

Comment:
This breaks building on boost < 1.68.

The diagnostic pragmas from above need to be moved as well.